docroot/modules/custom/foia_migrate/src/data-groomer.php

```diff
@@ -130,25 +144,42 @@ function set_component_id(&$component, &$id) {
  *   Type of personnel (e.g. foia_officer)
  * @param int $id
  *   Numerical ID to assign to FOIA personnel.
+ * @param object $misc
+ *   The Misc user object.
  *
  * @return mixed
  *   FOIA Personnel object with an assigned numerical ID if dealing with new
  *   personnel, FALSE otherwise.
  */
-function set_personnel_id(&$personnel, &$component, $personnel_type, &$id) {
-  $individual_personnel = $component->{$personnel_type};
-  unset($component->{$personnel_type});
-  $component->{$personnel_type} = new stdClass();
+function set_personnel_id(&$personnel, &$component, $personnel_type, &$id, $misc = NULL) {
+  if ($personnel_type === 'miscellaneous') {
+    $individual_personnel = $misc;
+  }
+  else {
+    $individual_personnel = $component->{$personnel_type};
+    unset($component->{$personnel_type});
+    $component->{$personnel_type} = new stdClass();
+  }
   $existing_id = '';
   if (isset($personnel['personnel'])) {
     $existing_id = get_existing_personnel_id($personnel['personnel'], $component, $individual_personnel);
   }
   if ($existing_id) {
-    $component->{$personnel_type}->id = $existing_id;
+    if ($personnel_type === 'miscellaneous') {
+      $component->miscellaneous->{$existing_id} = $existing_id;
```

Comment:
Definitely not a reason to hold back the PR, but would the script work if we modified this to be:
```
$component->miscellaneous->id[] = $existing_id;
```
Wondering if we can keep the data looking consistent. Right now we have:
```
            "service_center": {
                "id": 3
            },
            "miscellaneous": {
                "4": 4
            }
```
vs something like this:
```
            "service_center": {
                "id": 3
            },
            "miscellaneous": {
                "id": [
                  4
                ]
            }
```